content/blog/2019/09/2019-09-16-outreachy-audit-log-release.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Audit Log Plugin for Jenkins Releases 1.0"
+opengraph:
+  /images/logos/audit-log/jenkins-apache-logging.png
```

Comment:
```suggestion
  image: /images/logos/audit-log/jenkins-apache-logging.png
```

Reply:
Shame on me that I proposed an invalid syntax.  I was so proud of hacking together the image that I wrote bad syntax!  Thanks @zbynek for catching that mistake!